### DISCUSSION TITLE
Wipers ON by default on startup/wiper selector does to work.

### DISCUSSION THREAD

**Discussion by JuanWhite** [flybywiresim/aircraft#2298] [Help] [answered]
Hello!
I have the last dev version downloaded on 08dec2020 at 23z not sure of the version number...
When powering up the aircraft and wiper switch set into the off position wipers are ON. The wiper switch cannot be moved. Please Check the small video attached.

Thanks for your work.

https://gyazo.com/3d943d202e95cb67a2b6927b0d20d6d4

**Comment by Benjozork** [comment]
> 08nov2020

That is very old. Can you try with the current development version ?

**Reply by JuanWhite** [reply]
> > 08nov2020
> 
> That is very old. Can you try with the current development version ?

Sorry I meant December. Updated a couple hours back.

**Reply by Benjozork** [reply]
Getting more reports of this, standby

**Reply by nistei** [reply]
#2243 got merged today. Definitely related.

**Reply by Benjozork** [reply]
Hey @JuanWhite, do you have the YourControls mod installed ?

**Reply by JuanWhite** [reply]
Yes I do, is that the problem?

**Reply by nyhmbo** [reply]
yourcontrols can cause issues, try without it.

**Reply by Benjozork** [reply]
I am being told YC causes the issue. Make sure to remove the folder from `Community/` and report back !

**Reply by JuanWhite** [reply]
Deleted YC from community and wipers are working back! Thanks for the help!

**Comment by ImenesFBW** [comment, marked as answer]
Was unable to reproduce with the latest dev. 

I could replicate it if I place "yourcontrols" mod in the community folder.  So check if you have that, or any other mods causing the conflict.

**Reply by pepperoni505** [reply]
During testing of that PR I had the same issue, though no one else could replicate it. I notified Sequal and hopefully he gets a fix out.

**Reply by pepperoni505** [reply]
A little update: YourControls was updated to support the wiper changes, which fixes this issue

**Reply by MT2570** [reply]
Experiencing issue with the left wiper in release 0.5.0? As soon as I turn on the battery the wiper starts functioning although the switch is in the 'off' position. I cannot manipulate either of the wiper switches on the overhead. Tried with YourControls in the community folder and deleting it too, issue still persists.

**Reply by KingAsLan777** [reply]
Yeah, I have been having the same issues too, the left wiper keeps on going it won't turn off.

**Comment by pepperoni505** [comment]
Similar to what @ImenesFBW said, try updating or removing YourControls. There is a PR that fixes another issue with wipers that hasn't been merged yet. Once it has, all the wiper issues should be fixed.

**Reply by pepperoni505** [reply]
The issue has now been fixed in the latest master.

**Reply by monkeh88** [reply]
the wipers are still on , the switch doesn't work

**Reply by pepperoni505** [reply]
> the wipers are still on , the switch doesn't work

What version are you on? Also try removing other mods in your community folder
